requirements-cli.txt

```diff
@@ -1,2 +1,2 @@
 toml==0.10.2; (python_version >= "2.6" and python_full_version < "3.0.0") or (python_full_version >= "3.3.0")
-typing-extensions==4.0.1; python_version >= "3.6"
+typing-extensions==4.2.0; python_version >= "3.7"
```

Comment:
And here, or no?

Reply:
Requirements files aren't to be updated manually. They're created from the poetry lock file when propagate is ran.